conans/model/layout.py

```diff
@@ -131,3 +136,9 @@ def base_imports(self):
 
     def set_base_imports(self, folder):
         self._base_imports = folder
+
+    def set_base_export(self, folder):
+        self._base_export = folder
+
+    def set_base_export_sources(self, folder):
+        self._base_export_sources = folder
```

Comment:
Seems a getter for this, so we avoid the above (in class ConanFile) ``return self.folders._base_export_sources``, accesing a private member.

Or maybe avoid the setters, getters, make them public attributes?